docs/src/crossover.md

```diff
@@ -58,6 +58,12 @@ OX2
 POS
 ```
 
+Combinatorial crossovers:
+
+```@docs
+SXO
```

Comment:
There is a section for binary crossovers, in the beginning  of the file. I do not think you need to put it in the dedicated section.

Reply:
gotcha